### ISSUE TITLE
Update vignette for NBN

### ISSUE THREAD

**Issue by fozy81** [BiologicalRecordsCentre/NBN4R#7]
The README file has been updated to use NBN4R but the vignette has not. The README could also do with more work and could do with a real world example and perhaps more examples in general. Not sure the README and the vignette should mirror each other. Possibly best to have things in once place (vignette) and keep the readme basic info only and link to further info.

**Comment by AugustT** [comment]
I agree, I think the readme should focus on what the package does and how to install, vignette should be much more in depth
